elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -3,36 +3,81 @@
+                {{{feedback}}}
+            </div>
+            {{/display_feedback}}
+
+            {{#display_score_badge}}
+                {{#correct}}
+                    <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>
+                {{/correct}}
+                {{#incorrect}}
+                    <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>
+                {{/incorrect}}
+                {{#partial}}
+                    <span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i></span>
+                {{/partial}}
+            {{/display_score_badge}}
         </div>
-        {{/display_feedback}}
-
-        {{#display_score_badge}}
-            {{#correct}}
-                <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>
-            {{/correct}}
-            {{#incorrect}}
-                <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>
-            {{/incorrect}}
-            {{#partial}}
-                <span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i></span>
-            {{/partial}}
-        {{/display_score_badge}}
+    {{/answers}}
+{{/radio}}
+{{^radio}}
+<script>
+    $(function(){
+        $(document).ready(function(){
+            $(".pl-dropdown-choice-{{uuid}}.active").each(function(){
+                $("#pl-multiple-choice-dropdown-button-{{uuid}}").text($(this).text());
+            });
+        });
+
+        $(".pl-dropdown-choice-{{uuid}}").click(function(){
+            // First, set dropdown button to display same text as selected button
+            $("#pl-multiple-choice-dropdown-button-{{uuid}}").text($(this).text());
+            
+            // Next, change active choice to be what the user just selected
+            $(".pl-dropdown-choice-{{uuid}}").each(function(){
+                $(this).removeClass("active");
+            });
+            $(this).addClass("active");
+
+            // Finally, set input value
+            $("#pl-dropdown-input-{{uuid}}").val($(this).attr("value"));
+        });
+    });
+</script>
+<div class="dropdown py-1 d-inline-flex">
+    <button class="btn btn-secondary dropdown-toggle" type="button" id="pl-multiple-choice-dropdown-button-{{uuid}}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    </button>
+    <input type="hidden" name="{{name}}" id="pl-dropdown-input-{{uuid}}" {{^editable}}disabled{{/editable}}>
+    <div class="dropdown-menu" id="pl-multiple-choice-dropdown-menu-{{uuid}}" aria-labelledby="dropdownMenuButton">
+        {{#answers}}
+            <li><a class="dropdown-item pl-dropdown-choice-{{uuid}} {{#checked}}active{{/checked}}" value="{{key}}" {{^editable}}disabled{{/editable}}>
+                <div>{{^hide_letter_keys}}({{key}}){{/hide_letter_keys}} {{{html}}}</div>
+            </a></li>
```

Comment:
Any reason why you're using `li` elements here?

Reply:
Whoops, was copying and pasting too quickly.